### PR TITLE
Changes for version 2.0.0

### PR DESCRIPTION
**Pull request by abravosuse**
### PR creator: Description

The goal to this PR is to adapt the documentation to Trento version 2.0.0. The new version comes with a new checks engine that  no longer requires SSH to execute checks and that uses a message server to communicate with the agent. 

The PR can not be merged until the new version is actually released.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### CODE REVIEW

**Review by abravosuse — COMMENTED** [review]
Thank you @tomschr !

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -689,12 +629,12 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
           Open the configuration file <filename>/etc/trento/agent.yml</filename>
           and uncomment (use <literal>#</literal>) the entries for
-          <parameter>ssh-address</parameter>, <parameter>server-url</parameter> and
+          <parameter>facts-service-url</parameter>, <parameter>server-url</parameter> and
           <parameter>api-key</parameter>. Update the values appropriately:
         </para>
         <itemizedlist>
           <listitem>
-            <para><parameter>ssh-address</parameter>: local IP address</para>
+            <para><parameter>facts-service-url</parameter>: the address of the AMQP service shared with the checks engine, where fact gathering requests are received.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:637]: This AMQP service is new. I have some questions:

* I guess it's an acronym. What's the complete, resolved name of this acronym?
* This service sounds like an additional requirement to me. Shouldn't we add it somewhere in section "3 Requirement"?
* Should we describe this service a bit more? Or can we reference/link it to existing documentation?

**Review by tomschr — COMMENTED** [review]
@abravosuse Thank you Alberto! All fine, I just have one thing which needs perhaps some clarification.

File: trento/xml/article_sap_trento.xml (hunk 1 above)
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:637]: > I guess it's an acronym. What's the complete, resolved name of this acronym?
AMQP stands for Advanced Message Queuing Protocol.

> This service sounds like an additional requirement to me. Shouldn't we add it somewhere in section "3 Requirement"?
It's not a requirement as it is built-in in the new server architecture. AMQP is an application layer protocol that lets client applications (the agents) talk to the server (the new checks engine - WANDA) and interact. AMQP is the core protocol for RabbitMQ, a Message Broker that is now part of the Trento Server architecture allowing communication between the agents and the checks engine. 

> Should we describe this service a bit more? Or can we reference/link it to existing documentation?
This is transparent for the user. We didn't talk about the architecture of the old checks engine (RUNNER). I don't think there is a need to talk about the architecture of the new checks engine (WANDA) either.

Comment (tomschr) [reply trento/xml/article_sap_trento.xml:637]: Ahh, ok. That makes sense. Thanks for the explanation.

In that case I'd suggest to not only have the acronym but also the meaning. Something like this:

```
the AMQP (Advanced Message Queuing Protocol) service...
```

**Comment by abravosuse** [comment]
@tomschr I have added a new commit to the PR indicating the minimum length of the admin password. If you approve, you can merge the PR into main and publish.

**Review by tomschr — APPROVED** [review]
Looks good! Thanks Alberto! :+1:

**Pull request merged**